lms/djangoapps/verify_student/models.py

```diff
@@ -141,6 +141,34 @@ def active_at_datetime(self, deadline):
         )
 
 
+class ManualVerification(IDVerificationAttempt):
+    """
+    Each ManualVerification represents an enterprise user's verification that bypasses the need for
```

Comment:
nit: I think we should not tie this verification mode to only enterprise learners so we can remove the word `enterprise`.